Q1.java

```diff
@@ -0,0 +1,161 @@
+import java.util.*;
+
+class Q1
+{
+	public static void main(String[] args)
+	{
```

Comment:
Nit, but FYI: Most Java coding standards have the { on the same line as the method. For example:
Oracle: http://www.oracle.com/technetwork/java/codeconventions-150003.pdf
Google: https://google.github.io/styleguide/javaguide.html
Twitter: https://github.com/twitter/commons/blob/master/src/java/com/twitter/common/styleguide.md

By writing your methods this way you look like an inexperienced Java programmer who is more used to C/C#.
It might be worth reading some of the style guides, and fit into the conventions of whatever language you are using, then you'll "sound like a native speaker"